cluster/providers/openstack/setup.sh

```diff
@@ -4,15 +4,21 @@ set -ex
 
 export ext_ip=$(ip route get 8.8.8.8 | awk '{ print $7 }' | head -1)
 # set the EXTERNAL_IP env for the packstack container,its needed for accessing the NFS.
-sed -i "s/<set_external_ip>/${ext_ip}/g" cluster/providers/openstack/packstack_deployment.yml
+sed -i "s/<set_external_ip>/${ext_ip}/g" cluster/providers/openstack/manifests/packstack_deployment.yml
 
-kubectl apply -f cluster/providers/openstack/packstack_deployment.yml
+kubectl apply -f cluster/providers/openstack/manifests/packstack_deployment.yml
 while ! kubectl get deployment -n konveyor-forklift packstack; do sleep 10; done
 kubectl wait deployment -n konveyor-forklift packstack --for condition=Available=True --timeout=280s
 
 # deploy csi-driver-nfs
 cluster/providers/utils/deploy_csi_driver_nfs.sh "${ext_ip}"
 
+# apply openstack volume populator crds
+kubectl apply -f cluster/providers/openstack/manifests/osp-volume-populator-crd.yaml
```

Comment:
should be a TODO here to remove. once the deployment will work using forklift no need to apply any roles/CRD related to OSP populator.